libs/designsystem/src/lib/components/segmented-control/segmented-control.component.scss

```diff
@@ -32,6 +33,10 @@
       }
     }
   }
+  &.compact {
+    //TODO: verify if this is OK. It enables focus ring to not be cut off for first chip
```

Comment:
Handle this TODO. Probably a quick talk with @MadsBuchmann.

Reply:
What's the status on this TODO?

Reply:
Removed, we'll move forward with this implementation!